projects/exui-common-lib/src/lib/gov-ui/components/gov-uk-date/gov-uk-date.component.scss

```diff
@@ -0,0 +1,11 @@
+#date-input-container {
```

Comment:
@johnbenjamin-hmcts - as above, you might need to change the CSS selector depending on what you do with the `date-input-container` ID above.